job_board/views.py

```diff
@@ -1,3 +1,5 @@
-# from django.shortcuts import render
+from django.shortcuts import render
 
-# Create your views here.
+
+def job_board(request):
+    return render(request, 'job_board/job_board.html')
```

Comment:
Please pass a context dictionary to the rendered page so that the page title will update.
```py
def job_board(request):
    context = {'title': 'Job Board'}
    return render(request, 'job_board/job_board.html', context)
```

Reply:
Looks good, could be marked as resolved